utils/middleware/logging-context-middleware.ts

```diff
@@ -0,0 +1,22 @@
+import middy from "@middy/core";
+import { APIGatewayProxyResult } from "aws-lambda";
+import { CommonMiddlewareInputs } from "./common";
+import { logger } from "../logging";
+
+// Create the middleware and export it
+export const LoggingContextMiddleware = (): middy.MiddlewareObj<CommonMiddlewareInputs, APIGatewayProxyResult> => {
```

Comment:
Nice improvement.  Appreciate the more general nature of this logging middleware vice the IP logging middleware's console output.